spaces/aragon/index.json

```
@@ -1,29 +1,31 @@
 {
   "name": "Aragon",
   "network": "1",
-  "symbol": "ANT",
+  "symbol": "ANJ",
   "skin": "aragon",
   "domain": "gov.aragon.org",
   "strategies": [
     {
       "name": "erc20-balance-of",
       "params": {
-        "address": "0xa117000000f279D81A1D3cc75430fAA017FA5A2e",
-        "symbol": "ANT",
+        "address": "0xcD62b1C403fa761BAadFC74C525ce2B51780b184",
+        "symbol": "ANJ",
         "decimals": 18
       }
     },
     {
-      "name": "balancer",
+      "name": "anj-staked",
       "params": {
-        "address": "0xa117000000f279D81A1D3cc75430fAA017FA5A2e",
-        "symbol": "ANT BPT"
+        "jurorsRegistry": "0x0F7471C1df2021fF45f112878F755aAbe7AA16bF",
+        "symbol": "ANJ RGT",
```

```suggestion
        "symbol": "sANJ",
```

Perhaps this would look better, it's also what newly staked ANTv2 will use ("sANT")

---

Ah, this definitely looks fancier. 😄